lib/bolt/config.rb

```diff
@@ -187,6 +178,12 @@ def validate
         raise Bolt::CLIError, "Unsupported format: '#{self[:format]}'"
       end
 
+      if self[:transports][:ssh][:sudo_password] && self[:transports][:ssh][:run_as].nil?
+        logger = Logger.get_logger
+        logger.warn("'--sudo-password will not be used without specifying a " \
```

Comment:
Should `--sudo-password` be wrapped in single-quotes? There's a starting single-quote but not ending.

Reply:
we don't do that a few lines below so I'll  just remove it

Reply:
we don't quote a flag a few lines down so I removed it